plugins/hls-tactics-plugin/src/Ide/Plugin/Tactic/Judgements.hs

```diff
@@ -218,6 +218,8 @@ filterSameTypeFromOtherPositions dcon pos jdg =
    in disallowing Shadowed (M.keys to_remove) jdg
 
 
+{- HLINT ignore getAncestry "Replace case with maybe" -}
+
```

Comment:
Can I ask why this is disabled?

Reply:
I don't remember what I was thinking. The version with maybe looks fine to me now. Fixed.